browser/src/canvas/sections/ShapeHandleRotationSubSection.ts

```diff
@@ -0,0 +1,119 @@
+/* global Proxy _ */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+class ShapeHandleRotationSubSection extends HTMLObjectSection {
+	documentObject: boolean = true;
+
+	constructor (parentHandlerSection: ShapeHandlesSection, sectionName: string, size: number[], documentPosition: cool.SimplePoint, ownInfo: any) {
+        super(sectionName, size[0], size[1], documentPosition, null, true);
+
+        this.sectionProperties.parentHandlerSection = parentHandlerSection;
+		this.sectionProperties.ownInfo = ownInfo;
+		this.sectionProperties.mouseIsInside = false;
+		this.sectionProperties.previousCursorStyle = null;
+		this.sectionProperties.lastDraggingDistance = null;
+
+		this.getHTMLObject().style.borderRadius = '50%';
+		this.getHTMLObject().style.border = '1px solid black';
+		this.getHTMLObject().remove();
+		document.getElementById('map').appendChild(this.getHTMLObject());
+		this.mirrorEventsFromSourceToCanvasSectionContainer(this.getHTMLObject());
+	}
+
+	mirrorEventsFromSourceToCanvasSectionContainer (sourceElement: HTMLElement): void {
```

Comment:
this mirror thing possibly could be shared inside some base class

Reply:
Yes, i didn't want to do that now because we will remove them all :) Thank you for checking.